README.md

```diff
@@ -40,7 +40,7 @@ Here is the business story:
 
 During Sprint Planning, you estimated the story to be between 1 and 3 story points (in general, this corresponds to a few hours to a couple of days). We have committed to complete this by the end of our Sprint which ends soon. Please fork the code and submit a link to your fork with your changes when you have completed the assignment.
 
-The existing code is "legacy", and, ugh, it's ugly. Feel free to make any changes to the code as long as everything still works correctly and all specs pass. (Although, note that Awards are used elsewhere in the system, so their public interface must remain backward compatible).
+The existing code is “legacy”, and, ugh, it’s ugly. We’re attempting to move the legacy code toward a more object-oriented approach, and this would could be a good time to refactor. Feel free to make any changes to the code as long as everything still works correctly and all specs pass.
```

Comment:
I think you mean "could" instead of "would could".

Reply:
Well, _I_ think he means "would" instead of "would could" 😉

Reply:
Team Could

Reply:
FWIW, I agree with @gretchenziegler. We _should_ err on the side of being explicit.

Reply:
I've made the case elsewhere for leaving the slightest ambiguity so you probably already knew where I *stood*. If the folks closest to the hiring feel strongly - I trust you - do with it what you *would*. Either way you go, I'm *good*.

Reply:
I'm changing it to would. Consider this conversation a helpful easter egg for intrepid candidates.